examples/workdir/env/extravars

```diff
@@ -6,4 +6,8 @@ ceph_user_keyring_b64key: QVFDZWoySmNBQUFBQUJBQWlhSTZJZEw2Nml4S1FINEVlOTBTbXc9PQ
 suse_osh_deploy_ceph_mons: ['192.168.50.10:6789']
 # This is the VIP that will be used for the openstack deployment on kubernetes
 # (ingress controller)
-suse_osh_deploy_vip_with_cidr: 192.168.50.46/24
+# Deployment goal is either 'airship' or 'osh'
+socok8s_deployment_goal: airship
```

Comment:
Two things here:
1) If this PR is about common naming, we shouldn't add this variable here, but in a later commit.
2) Why does this variable exist? I fail to see the grand scheme of things for multiple reasons: A) deployers can run the appropriate code by running the right commands (see docs). B) It will be more complex and error prone for users. C) This variable is not documented.

Can you remove this variable, and the variables in L12-L13?

Reply:
My thinking was that this flag may help in making 'state' decisions what kind of deployment is targeted or current type of deployment is already orchestrated. 
If we are just doing deploy_osh or deploy_airship , then commands itself are obvious . I was thinking it may be useful when we are doing peacemeal actions (subset of 7 or 8 or 1-6 actions) e,g. creating subvolumes when adding compute (lifecycle steps) or creating inventory file for osh or airship deployment . 
IMHO, i am sure there might be better way or too premature at current state of things. Just added more as placeholder and can be removed easily if its not needed as post-merge cleanup.

Reply:
Well I kinda agree and don't agree.

I agree on the fact that it's maybe too premature to call on the state of things. But then, if it's premature, why creating a new variable on some predictions of things that are premature to tell?

The addition of this var should be in a subsequent commit.

Reply:
Okay..will remove it as currently its not used anywhere.

Reply:
Its already removed so resolving it again